### DISCUSSION TITLE
Option to increase thickness of Camera2D limits

### DISCUSSION THREAD

**Discussion by valkyrienyanko** [godotengine/godot-proposals#5333] [2D]
The camera boundary is hard to see on a project that uses 64 x 64 tilemaps. Having the option to increase the thickness of this line would make it a lot more visible. I know I can change the color from yellow to something else like black but I like having it as yellow.

![image](https://user-images.githubusercontent.com/6277739/188328908-1360b899-5dd1-4c95-99f7-af3fd7e8ae4a.png)

**Comment by Calinou** [comment]
> I know I can change the color from yellow to something else like black but I like having it as yellow.

Instead of adding a new setting (and making the already lengthy Editor Settings even longer), I think we should draw an outline instead to make the line visible on any background. This can be done by drawing a yellow line, then a black line with a slightly different offset.

This can also be done for grid lines, the X/Y axes and so on. (Less opaque lines will also use less opaque shadows.)
